atom/browser/atom_permission_manager.cc

```diff
@@ -237,6 +244,9 @@ bool AtomPermissionManager::CheckPermissionWithDetails(
     const GURL& requesting_origin,
     const base::DictionaryValue* details) const {
   if (check_handler_.is_null()) {
+    if (Browser::Get()->secure_mode_enabled()) {
+      return false;
+    }
```

Comment:
_(minor)_ 247-250 could be rephrased as `return Browser::Get()->secure_mode_enabled();`

Reply:
@MarshallOfSound what do you think? I wanted to make an explicit fork in the code for the secure mode to make the behavior clearly visible.

Reply:
As I alluded to below in a different comment for this particular API I would greatly appreciate the explicitness this code fork brings.  It makes it really clear what is going to happen when secure mode is enabled.